README.md

```diff
@@ -58,7 +58,7 @@ go get github.com/moov-io/iso8583
 
 ### Define your specification
 
-Currently, we support following ISO 8583 specifications:
+Currently, we have defined the following ISO 8583 specifications:
```

Comment:
This was a little confusing for myself and my team. We assumed by "support" we needed to add a new spec to support anything not based off of the 1987 spec. That's incorrect. The library provides some sample specs already, but there is support for newer versions of the spec.